docs/docs/anatomy.md

```diff
@@ -148,6 +163,8 @@ export class Service {
 <details>
 <summary>See the `../equinox-config.ts` snippet</summary>
 
+Feel free to copy-paste this snippet for your own project.
```

Comment:
Maybe mention that in general you have one set of this binding per repo ?
And that in most real apps you'll typically have bindings for one concrete store and the memory store

Reply:
modified